ddtrace/monkey.py

```diff
@@ -154,18 +160,20 @@ def patch(raise_errors=True, **patch_modules):
     modules = [m for (m, should_patch) in patch_modules.items() if should_patch]
     for module in modules:
         if module in _PATCH_ON_IMPORT:
-            # If the module has already been imported then patch immediately
-            if module in sys.modules:
-                patch_module(module, raise_errors=raise_errors)
-
-            # Otherwise, add a hook to patch when it is imported for the first time
-            else:
-                # Use factory to create handler to close over `module` and `raise_errors` values from this loop
-                when_imported(module)(_on_import_factory(module, raise_errors))
-
-                # manually add module to patched modules
-                with _LOCK:
-                    _PATCHED_MODULES.add(module)
+            modules_to_poi = _PATCH_ON_IMPORT[module]
+            for m in modules_to_poi:
```

Comment:
Why are we looping through `modules_to_poi` here? Correct me if I'm wrong, but shouldn't line 163 yield a single module?

Reply:
The values of the `_PATCH_ON_IMPORT` dictionary should be tuples of strings